### PR TITLE
fix: keyboard controller should attach event on document

### PR DESCRIPTION
**Pull request by doouding**

### CODE REVIEW

**Comment by doouding** [comment]
* **#8801** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/8801?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/8801?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `master`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: packages/framework/block-std/src/gfx/keyboard.ts [hunk 1]
```diff
@@ -16,27 +16,35 @@ export class KeyboardController {
...
+    event: 'keydown' | 'keyup',
+    callback: (keyboardEvt: KeyboardEvent) => void
+  ) {
+    document.addEventListener(event, callback, false);
+
+    return () => {
+      document.removeEventListener(event, callback, false);
+    };
+  }
+
```
Comment (L-Sun) [inline packages/framework/block-std/src/gfx/keyboard.ts:47]: Does listening on the document affect user interactions on the AFFiNE side?

Comment (doouding) [reply packages/framework/block-std/src/gfx/keyboard.ts:47]: I'm not sure. It should be fine.

Comment (Flrande) [reply packages/framework/block-std/src/gfx/keyboard.ts:47]: Have you tested the situation where multiple editors exist at the same time?

Comment (doouding) [reply packages/framework/block-std/src/gfx/keyboard.ts:47]: I've added an e2e test

**Review by doodlewind — APPROVED** [review]

**Pull request merged**